certbot/certbot/_internal/cli/__init__.py

```diff
@@ -95,6 +95,11 @@ def prepare_and_parse_args(plugins: plugins_disco.PluginsRegistry, args: List[st
         action="store_true", default=flag_default("preconfigured_renewal"),
         help=argparse.SUPPRESS
     )
+    helpful.add(
+        None, "--issuance-timeout", type=int,
```

Comment:
I have a feeling that we should make this flag appear in the `--help` topics for `all`, `certonly` and `run`:

```suggestion
        [None, "certonly", "run"], "--issuance-timeout", type=int,
```